layouts/partials/head.html

```diff
@@ -37,13 +37,23 @@
     <!-- SEO canonicals: helps Google understand your site better when using marketing campaign tagging etc -->
     <link rel="canonical" href="{{ .Permalink }}" />
 
-    <link rel="stylesheet" type="text/css" href="{{ .Site.BaseURL }}/css/normalize.min.css" media="print" onload="this.media='all'">
-    <link rel="stylesheet" type="text/css" href="{{ .Site.BaseURL }}/css/main.css">
-    <link disabled id="dark-theme" rel="stylesheet" href="{{ .Site.BaseURL }}/css/dark.css">
+    {{ $cssNormalize := resources.Get "css/normalize.min.css" }}
+    <link rel="stylesheet" type="text/css" href="{{ $cssNormalize.RelPermalink }}" media="print" onload="this.media='all'">
 
-    <script src="{{ .Site.BaseURL }}/js/svg-injector.min.js"></script>
-    <script src="{{ .Site.BaseURL }}/js/feather-icons.min.js"></script>
-    <script src="{{ .Site.BaseURL }}/js/main.js"></script>
+    {{ $cssMain := resources.Get "css/main.css" | minify }}
+    <link rel="stylesheet" type="text/css" href="{{ $cssMain.RelPermalink }}" onload="this.media='all'">
+
+    {{ $cssDark := resources.Get "css/dark.css" | minify }}
+    <link id="dark-theme" rel="stylesheet" href="{{ $cssDark.RelPermalink }}" onload="this.media='all'">
```

Comment:
Let's not load any CSS files asynchronously. We can get rid of `onload="this.media='all'`. Running into FOUC or visual glitches is far worse than saving a few miliseconds IMO.

Reply:
Removed it